src/main/java/com/google/cloud/tools/managedcloudsdk/command/AsyncStreamHandlerFactory.java

```diff
@@ -16,32 +16,32 @@
 
 package com.google.cloud.tools.managedcloudsdk.command;
 
-import com.google.cloud.tools.managedcloudsdk.MessageListener;
+import com.google.cloud.tools.managedcloudsdk.ConsoleListener;
 import com.google.cloud.tools.managedcloudsdk.process.AsyncStreamHandler;
 import com.google.common.base.Charsets;
 
 /** Factory to create default implementations of {@link AsyncStreamHandler}. */
 class AsyncStreamHandlerFactory {
 
   /**
-   * Create a new AsyncStreamHandler using the {@link MessageListenerForwardingHandler}
+   * Create a new AsyncStreamHandler using the {@link ConsoleListenerForwardingByteHandler}
    * implementation.
    */
-  AsyncStreamHandler newHandler(MessageListener messageListener) {
-    return new AsyncByteConsumer(new MessageListenerForwardingHandler(messageListener));
+  AsyncStreamHandler newHandler(ConsoleListener consoleListener) {
+    return new AsyncByteConsumer(new ConsoleListenerForwardingByteHandler(consoleListener));
   }
 
-  static class MessageListenerForwardingHandler implements ByteHandler {
+  static class ConsoleListenerForwardingByteHandler implements ByteHandler {
 
-    private final MessageListener messageListener;
+    private final ConsoleListener consoleListener;
 
-    MessageListenerForwardingHandler(MessageListener messageListener) {
-      this.messageListener = messageListener;
+    ConsoleListenerForwardingByteHandler(ConsoleListener consoleListener) {
+      this.consoleListener = consoleListener;
     }
 
     @Override
     public void bytes(byte[] bytes, int length) {
-      messageListener.message(new String(bytes, 0, length, Charsets.UTF_8));
+      consoleListener.console(new String(bytes, 0, length, Charsets.UTF_8));
```

Comment:
nit: `0, length, ` can be omitted

Reply:
The content doesn't necessarily occupy the full array, it's a fixed buffer that's reused, so we need length in here.